--- Running scipy.sparse.linalg.minres#42NN/minresNN, strategy: relancer
[ERR MSG]: OrderedDict([('project', 'scipy.sparse.linalg.minres#42NN'), ('notebook', 'minresNN'), ('text', '  File "minresNN.py", line 6, in <module>\n    (x, info) = minres(A=A, b=b, x0=x0, shift=0.0, tol=1e-05, maxiter=None, xtype=None, M=None)\nTypeError: minres() got an unexpected keyword argument \'xtype\'\n')])
  File "minresNN.py", line 6, in <module>
Error inducing API short name: , line no: 6
Error inducing API fqn: scipy.sparse.linalg.minres, line no: 6
*** TypeError: PLACEHOLDER
Repair Action: arg_name
Key Params: ['A', 'b', 'x0', 'shift', 'tol', 'maxiter', 'xtype', 'M']
Prev Params: None
Prev Repair Action: None
Line No: 6
Prev Line No: 0
GitHub Candidates:
[]
KP:['A', 'b', 'x0', 'shift', 'tol', 'maxiter', 'xtype', 'M']
EXPP: ['A', 'b', 'x0', 'shift', 'tol', 'maxiter', 'xtype', 'M']
APIDoc Candidates:
[]
All Solution Candidates:
[]
[INFO] This case cannot be fixed!


real	0m0.531s
user	0m1.666s
sys	0m5.706s
